src/database/seeders/20191119054216-users.js

```diff
@@ -5,7 +5,9 @@ module.exports = {
     return queryInterface.bulkInsert('users', [{
         firstName: 'John',
         lastName: 'Doe',
-        email: 'demo@demo.com',
+        email: 'demo@gmail.com',
+        isVerified:'false',
+        signupType:'normal',
```

Comment:
@hezronkimutai  because isVerified is of Boolean type, you don't need to put a single quote.

Reply:
signupType should be 'Barefoot' instead of Normal, just the sake of being specific.

Reply:
Yeah, exactly, thanks Victor.